### ISSUE TITLE
Review comment has white background w/ Refined Github

### ISSUE THREAD

**Issue by MCOfficer** [StylishThemes/GitHub-Dark#809]
* **Browser**:
Firefox 64.0
* **Operating System**:
Ubuntu MATE 18.04
* **Screenshot**:
![image](https://user-images.githubusercontent.com/22377202/50519381-edb88b00-0aba-11e9-94f0-14b4e3d48135.png)

* **HTML of the section where the issue occurs**:
<details>
 <summary>Beware, it's a lot</summary>

````html<div class="review-comment js-minimizable-comment-group js-targetable-comment" id="discussion_r244336658" data-gid="MDI0OlB1bGxSZXF1ZXN0UmV2aWV3Q29tbWVudDI0NDMzNjY1OA==" data-url="/_render_node/MDI0OlB1bGxSZXF1ZXN0UmV2aWV3Q29tbWVudDI0NDMzNjY1OA==/comments/review_comment">

    <div class="minimized-comment d-none">
        

  <details class="Details-element details-reset " data-body-version="69352332f09a96826f1ccabb2d215c8a">
    <summary class="text-gray f6">
      <div class="d-flex flex-justify-between flex-items-center">
        <h3 class="review-comment-contents bg-white f5 text-normal text-italic" style="margin-left:38px">
          <div class="discussion-item-icon discussion-item-icon-gray text-gray">
            <svg class="octicon octicon-fold" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7 9l3 3H8v3H6v-3H4l3-3zm3-6H8V0H6v3H4l3 3 3-3zm4 2c0-.55-.45-1-1-1h-2.5l-1 1h3l-2 2h-7l-2-2h3l-1-1H1c-.55 0-1 .45-1 1l2.5 2.5L0 10c0 .55.45 1 1 1h2.5l1-1h-3l2-2h7l2 2h-3l1 1H13c.55 0 1-.45 1-1l-2.5-2.5L14 5z"></path></svg>
          </div>
          <div class="discussion-item-copy d-inline-block">
                This comment has been minimized.

          </div>
        </h3>
        <div class="Details-content--closed btn-link text-gray"><svg class="octicon octicon-unfold mr-1" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M11.5 7.5L14 10c0 .55-.45 1-1 1H9v-1h3.5l-2-2h-7l-2 2H5v1H1c-.55 0-1-.45-1-1l2.5-2.5L0 5c0-.55.45-1 1-1h4v1H1.5l2 2h7l2-2H9V4h4c.55 0 1 .45 1 1l-2.5 2.5zM6 6h2V3h2L7 0 4 3h2v3zm2 3H6v3H4l3 3 3-3H8V9z"></path></svg>Show comment</div>
        <div class="Details-content--open btn-link text-gray"><svg class="octicon octicon-fold mr-1" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7 9l3 3H8v3H6v-3H4l3-3zm3-6H8V0H6v3H4l3 3 3-3zm4 2c0-.55-.45-1-1-1h-2.5l-1 1h3l-2 2h-7l-2-2h3l-1-1H1c-.55 0-1 .45-1 1l2.5 2.5L0 10c0 .55.45 1 1 1h2.5l1-1h-3l2-2h7l2 2h-3l1 1H13c.55 0 1-.45 1-1l-2.5-2.5L14 5z"></path></svg>Hide comment</div>
      </div>
    </summary>
    <div class="py-2 pl-6 pr-0">
      <div class="previewable-edit  js-task-list-container reorderable-task-lists">
        <div class="edit-comment-hide">
          <div class="timeline-comment-actions">
            
















<details class="details-overlay details-reset position-relative d-inline-block js-dropdown-details " id="">
  <summary class="btn-link timeline-comment-action" aria-haspopup="true" aria-label="Show options">
    <svg aria-label="Show options" class="octicon octicon-kebab-horizontal" viewBox="0 0 13 16" version="1.1" width="13" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M1.5 9a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3zm5 0a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3zM13 7.5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0z"></path></svg>
  </summary>
  <div class="dropdown-menu dropdown-menu-sw show-more-popover text-gray-dark anim-scale-in" role="menu">
        <clipboard-copy class="dropdown-item btn-link" for="discussion_r244336658-minimized-permalink" aria-label="Copy link" data-toggle-for="details-discussion_r" role="menuitem" tabindex="0">
    Copy link
  </clipboard-copy>

        <button type="button" class="dropdown-item btn-link d-none js-comment-quote-reply" data-toggle-for="details-discussion_r">
    Quote reply
  </button>

      
  </div>
</details>

          </div>
            <a class="float-left mt-1" data-hovercard-type="user" data-hovercard-url="/hovercards?user_id=17688683" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="/Amazinite" aria-describedby="hovercard-aria-description"><img class="avatar" alt="@Amazinite" src="https://avatars3.githubusercontent.com/u/17688683?s=60&amp;v=4" width="28" height="28"></a>
          <div class="review-comment-contents">
            <h4 class="f5 text-normal d-inline text-gray-dark">
              <strong class="text-gray">
                

  <a class="author text-inherit css-truncate-target rgh-fullname" data-hovercard-type="user" data-hovercard-url="/hovercards?user_id=17688683" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="/Amazinite" aria-describedby="hovercard-aria-description">Amazinite</a>
  

              </strong>
              <span class="text-gray">
                  <a href="#discussion_r244336658" id="discussion_r244336658-minimized-permalink" class="timestamp"><relative-time datetime="2018-12-28T14:48:12Z" title="Dec 28, 2018, 3:48 PM GMT+1">23 minutes ago</relative-time></a>
              </span>
            </h4>
            
    <span class="timeline-comment-label tooltipped tooltipped-multiline tooltipped-s" aria-label="This user has been invited to collaborate on the endless-sky repository.">
      Collaborator
    </span>

            <task-lists disabled="" sortable="">
              <div class="comment-body markdown-body p-0 pt-1 js-comment-body ">
                  <p>This doesn't seem to be the case.</p>
              </div>
            </task-lists>
          </div>
        </div>

      </div>
    </div>
  </details>


    </div>

  <div class="previewable-edit js-suggested-changes-container js-task-list-container unminimized-comment js-comment  reorderable-task-lists" data-body-version="69352332f09a96826f1ccabb2d215c8a" data-thread-side="right">
    <div class="edit-comment-hide">
      <div class="timeline-comment-actions">
          
  <details class="details-overlay details-reset position-relative d-inline-block js-socket-channel js-updatable-content js-dropdown-details js-reaction-popover-container js-comment-header-reaction-button" data-channel="reaction:pull-request-review-comment:244336658" data-url="/_render_node/MDI0OlB1bGxSZXF1ZXN0UmV2aWV3Q29tbWVudDI0NDMzNjY1OA==/comments/comment_header_reaction_button">
    <summary class="btn-link timeline-comment-action" aria-label="Add your reaction" aria-haspopup="true">
      <svg class="octicon octicon-plus-small add-reaction-plus-icon" viewBox="0 0 7 16" version="1.1" width="7" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 4H3v3H0v1h3v3h1V8h3V7H4V4z"></path></svg>
      <svg class="octicon octicon-smiley" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm4.81 12.81a6.72 6.72 0 0 1-2.17 1.45c-.83.36-1.72.53-2.64.53-.92 0-1.81-.17-2.64-.53-.81-.34-1.55-.83-2.17-1.45a6.773 6.773 0 0 1-1.45-2.17A6.59 6.59 0 0 1 1.21 8c0-.92.17-1.81.53-2.64.34-.81.83-1.55 1.45-2.17.62-.62 1.36-1.11 2.17-1.45A6.59 6.59 0 0 1 8 1.21c.92 0 1.81.17 2.64.53.81.34 1.55.83 2.17 1.45.62.62 1.11 1.36 1.45 2.17.36.83.53 1.72.53 2.64 0 .92-.17 1.81-.53 2.64-.34.81-.83 1.55-1.45 2.17zM4 6.8v-.59c0-.66.53-1.19 1.2-1.19h.59c.66 0 1.19.53 1.19 1.19v.59c0 .67-.53 1.2-1.19 1.2H5.2C4.53 8 4 7.47 4 6.8zm5 0v-.59c0-.66.53-1.19 1.2-1.19h.59c.66 0 1.19.53 1.19 1.19v.59c0 .67-.53 1.2-1.19 1.2h-.59C9.53 8 9 7.47 9 6.8zm4 3.2c-.72 1.88-2.91 3-5 3s-4.28-1.13-5-3c-.14-.39.23-1 .66-1h8.59c.41 0 .89.61.75 1z"></path></svg>
    </summary>
    
<div class="dropdown-menu dropdown-menu-sw add-reaction-popover js-add-reaction-popover anim-scale-in">
  <!-- '"` --><!-- </textarea></xmp> --><form class="reaction-popover-form js-pick-reaction" action="/users/endless-sky/reactions" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="✓"><input type="hidden" name="_method" value="put"><input type="hidden" name="authenticity_token" value="GSYo6SG76lpbEJ+sOXf+mvr2o7g32eSub7DWensxJ/vH5696jfOfH0DjR3QII4KIf10BFiHvPJgmRPbBomVVCQ==">
    <p class="text-gray mx-2 my-1">
      <span class="js-reaction-description">Pick your reaction</span>
      <img alt="" class="loading-spinner" src="https://github.githubassets.com/images/spinners/octocat-spinner-32.gif" width="16" height="16">
    </p>

    <div class="dropdown-divider"></div>

    <div class="add-reactions-options mx-1 mb-1">
      <input type="hidden" name="input[subjectId]" value="MDI0OlB1bGxSZXF1ZXN0UmV2aWV3Q29tbWVudDI0NDMzNjY1OA==">

        <button type="submit" class="btn-link add-reactions-options-item js-reaction-option-item" data-reaction-label="+1" name="input[content]" aria-label="React with thumbs up emoji" value="THUMBS_UP react">
          <g-emoji alias="+1" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png" class="emoji"><img class="emoji" alt="+1" src="https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png" width="20" height="20"></g-emoji>
        </button>
        <button type="submit" class="btn-link add-reactions-options-item js-reaction-option-item" data-reaction-label="-1" name="input[content]" aria-label="React with thumbs down emoji" value="THUMBS_DOWN react">
          <g-emoji alias="-1" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f44e.png" class="emoji"><img class="emoji" alt="-1" src="https://github.githubassets.com/images/icons/emoji/unicode/1f44e.png" width="20" height="20"></g-emoji>
        </button>
        <button type="submit" class="btn-link add-reactions-options-item js-reaction-option-item" data-reaction-label="Laugh" name="input[content]" aria-label="React with laugh emoji" value="LAUGH react">
          <g-emoji alias="smile" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" class="emoji"><img class="emoji" alt="smile" src="https://github.githubassets.com/images/icons/emoji/unicode/1f604.png" width="20" height="20"></g-emoji>
        </button>
        <button type="submit" class="btn-link add-reactions-options-item js-reaction-option-item" data-reaction-label="Hooray" name="input[content]" aria-label="React with hooray emoji" value="HOORAY react">
          <g-emoji alias="tada" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f389.png" class="emoji"><img class="emoji" alt="tada" src="https://github.githubassets.com/images/icons/emoji/unicode/1f389.png" width="20" height="20"></g-emoji>
        </button>
        <button type="submit" class="btn-link add-reactions-options-item js-reaction-option-item" data-reaction-label="Confused" name="input[content]" aria-label="React with confused emoji" value="CONFUSED react">
          <g-emoji alias="thinking_face" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/1f615.png" class="emoji"><img class="emoji" alt="thinking_face" src="https://github.githubassets.com/images/icons/emoji/unicode/1f615.png" width="20" height="20"></g-emoji>
        </button>
        <button type="submit" class="btn-link add-reactions-options-item js-reaction-option-item" data-reaction-label="Heart" name="input[content]" aria-label="React with heart emoji" value="HEART react">
          <g-emoji alias="heart" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/2764.png" class="emoji"><img class="emoji" alt="heart" src="https://github.githubassets.com/images/icons/emoji/unicode/2764.png" width="20" height="20"></g-emoji>
        </button>
    </div>
</form></div>

  </details>

        <details class="details-overlay details-reset position-relative d-inline-block js-dropdown-details" id="details-discussion_r244336658">
          <summary class="btn-link timeline-comment-action" aria-haspopup="true" aria-label="Show more options">
            <svg aria-label="Show more options" class="octicon octicon-kebab-horizontal" viewBox="0 0 13 16" version="1.1" width="13" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M1.5 9a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3zm5 0a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3zM13 7.5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0z"></path></svg>
          </summary>

          <div class="dropdown-menu dropdown-menu-sw show-more-popover text-gray-dark anim-scale-in">

            <clipboard-copy class="dropdown-item btn-link tooltipped tooltipped-nw" for="discussion_r244336658-permalink" aria-label="Copy link" data-toggle-for="details-discussion_r244336658" role="menuitem" tabindex="0">
              Copy link
            </clipboard-copy>

            <button type="button" class="dropdown-item btn-link d-none js-comment-quote-reply" data-toggle-for="details-discussion_r244336658">
              Quote reply
            </button>







          </div>
        </details>
      </div>

      <span class="float-left mt-1">
        <a class="d-inline-block" data-hovercard-type="user" data-hovercard-url="/hovercards?user_id=17688683" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="/Amazinite" aria-describedby="hovercard-aria-description"><img class="avatar" alt="@Amazinite" src="https://avatars3.githubusercontent.com/u/17688683?s=60&amp;v=4" width="28" height="28"></a>
      </span>

      <div class="review-comment-contents js-suggested-changes-contents" data-thread-side="right">
        <h4 class="f5 text-normal d-inline">
          <strong>
              <a class="author link-gray-dark rgh-fullname" data-hovercard-type="user" data-hovercard-url="/hovercards?user_id=17688683" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="/Amazinite" aria-describedby="hovercard-aria-description">Amazinite</a> 
          </strong>
          <span class="text-gray">

              <a href="#discussion_r244336658" id="discussion_r244336658-permalink" class="js-timestamp timestamp d-inline-block">
                <relative-time datetime="2018-12-28T14:48:12Z" title="Dec 28, 2018, 3:48 PM GMT+1">23 minutes ago</relative-time>
              </a>

          </span>
        </h4>

        
    <span class="timeline-comment-label tooltipped tooltipped-multiline tooltipped-s" aria-label="This user has been invited to collaborate on the endless-sky repository.">
      Collaborator
    </span>





        <task-lists disabled="" sortable="">
          <div class="comment-body markdown-body  js-comment-body">
            <p>This doesn't seem to be the case.</p>
          </div>
        </task-lists>


          

<div class="comment-reactions  js-reactions-container js-socket-channel js-updatable-content" data-channel="reaction:pull-request-review-comment:244336658" data-url="/_render_node/MDI0OlB1bGxSZXF1ZXN0UmV2aWV3Q29tbWVudDI0NDMzNjY1OA==/comments/reactions">
</div>

      </div>
    </div>

  </div>
</div>
````
</details>

<br><p>

[Here](https://github.com/endless-sky/endless-sky/pull/3761#discussion_r244336658)'s the review in question.

**Comment by MCOfficer** [comment]
> Im unable to duplicate this Windows side Firefox 64, is your GitHub Dark Style up-to-date? I think Ill try this Linux side just to see if I can dupe there.

For the record, I checked that the usercss is up to date, even reinstalled it. Might be some OS-shenanigans involved, but that's your metier.

**Comment by the-j0k3r** [comment]
I went with @Ash258 direct link

Linux Ubuntu 18.04 LTS (Not Mate) + Firefox 64 no dupe.
Windows + Goggle Chrome Version 71.0.3578.98 no dupe
Windows + Firefox dev 65.0b6 (64-bit) and
Opera 65.0b6 (64-bit) I get something weird but not the reported issue. (see screenshot below)

![white](https://user-images.githubusercontent.com/31389848/50520268-a97bb980-0abf-11e9-8e6e-13de8d00360f.PNG)

**Comment by Ash258** [comment]
@MCOfficer Do you use [Refined Github](https://github.com/sindresorhus/refined-github) extension?

When I disable all other github related things, it's not happening anymore.

I tracked it down to `comment:target` rule (https://github.com/sindresorhus/refined-github/blob/b745fe56b9671c7119ea56546ef23b724bca4697/source/content.css#L461-L463)

I am not sure how to handle this properly.

**Comment by xt0rted** [comment]
GitHub's been changing how they style selected comments. Instead of using the `:target` pseudo selector they're using the `aria-selected="true"` attribute along with some additional JS to toggle this attribute.

For the comment at https://github.com/endless-sky/endless-sky/pull/3761#discussion_r244336658 the page isn't adding the `aria-selected="true"` attribute on page load, this is why there's no blue border showing on it either. This does work on other pages though such as https://github.com/StylishThemes/GitHub-Dark/pull/806#discussion_r243036111.

If you click on the time stamp link the page does add the attribute and so it gets styled correctly. I've reported this issue to someone at GitHub and waiting to hear back about it.

In 71337016e43fd71de991e337b0a89e8ed77ac126 we had to remove the older `:target` selectors because they were interfering with the new attribute selectors and elements were being styled when they shouldn't be. In recent days GitHub removed the rest of those selectors from their stylesheet as well.

Refined GitHub is still using the `:target` selector so our styles aren't overriding their background color now. They did remove some of these in https://github.com/sindresorhus/refined-github/pull/1461 but didn't remove the ones at [content.css#L460-L477](https://github.com/sindresorhus/refined-github/blob/master/source/content.css#L460-L477).

The issue we're seeing right now is a combination of GitHub not adding the `aria-selected="true"` attribute in some instances, and Refined GitHub still using `:target` for their styles.  GitHub's issue aside, I'm inclined to say this is a bug in Refined GitHub now and not our style.
